webpack.config.js

```diff
@@ -94,7 +95,7 @@ module.exports = {
         analyzerMode: mode !== "development" ? "disabled" : "server",
       }),
     ].filter(Boolean),
-    devtool: dev && "inline-cheap-module-source-map",
+    devtool: sourceMap ? "inline-cheap-module-source-map" : "source-map",
```

Comment:
@mukhtyar this will still build source maps when setting the env variable to false